plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -109,6 +110,55 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    @Override
+    public void testAddColumn()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testAddColumn);
+    }
+
+    @Override
+    public void testDropColumn()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testDropColumn);
+    }
+
+    @Override
+    public void testAlterTableAddLongColumnName()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testAlterTableAddLongColumnName);
+    }
+
+    @Override
+    protected void testAddAndDropColumnName(String columnName, boolean delimited)
+    {
+        skipSchemaChangeOperationLockInterruptedError(() -> super.testAddAndDropColumnName(columnName, delimited));
+    }
+
+    @Override
+    protected String dataMappingTableName(String trinoTypeName)
+    {
+        return "test_data_mapping_smoke_" + trinoTypeName.replaceAll("[^a-zA-Z0-9]", "_") + randomNameSuffix();
+    }
+
+    private void skipSchemaChangeOperationLockInterruptedError(Runnable schemaChangeOperation)
```

Comment:
Why not use `@Flaky` annotation?

Reply:
I saw the annotation is marked with `Deprecated`, shall we use it?